### DISCUSSION TITLE
[feature request]drawing tablet

### DISCUSSION THREAD

**Discussion by jimichou-gm** [AppGround-io/bluetooth-keyboard-and-mouse-support#142] [Ideas]
is it possible to provide drawing tablet function? it would be great if my android phone can works as an expensive drawing tablet for PC.

**Comment by sky-why-testing** [comment]
Actually, it would be a very desirable idea
However
(1) need to register the device as stylus in bluetooth on your window 10 computer 
(2) send not only the location information, but also pressure sensitivity reading to the computer
